pytorch_lightning/utilities/apply_to_collection.py

```diff
@@ -0,0 +1,37 @@
+from collections import Mapping, Sequence
```

Comment:
probably it won't be the only function here, right? what about renaming it as `convertors` or any other suggestion because I see we have already convertors in metrics so not confuse users... :]

Reply:
I think `convertors` may be misleading, since this does not always convert, but you can for example also use this to simply multiply every scalar with a factor.
Unfortunately I wasn't able to find a better name, but any suggestions are welcome :)

Reply:
`handlers`, `apply_func`?

Reply:
I think `apply_func` is good